src/components/panel/panel.js

```diff
@@ -637,7 +660,9 @@ var FOCUS_TRAP_TEMPLATE = angular.element(
 
 
 /**
+ * @description
```

Comment:
Where do the @decription come from? These aren't ngdoc, like above where @decription matters.